onnxruntime/python/tools/transformers/benchmark_gpt2.py

```diff
@@ -301,14 +376,30 @@ def main():
                                             size=(batch_size, 1),
                                             dtype=torch.int64,
                                             device=device)
-            torch_latency, ort_latency = inference(model,
-                                                   session,
-                                                   dummy_input_ids,
-                                                   dummy_past,
-                                                   args.test_times,
-                                                   verify_outputs=args.validate_onnx)
+                        
+            # Calculate the expected output shapes
+            last_state_shape = [
+                batch_size, 1, config.hidden_size
+            ]
+            present_shape = [
+                2, batch_size, config.num_attention_heads, sequence_length + 1, 
+                int(config.hidden_size / config.num_attention_heads)
+            ]
+
+            latencies = inference(model,
+                                    session,
```

Comment:
format this script (see onnxruntime\ReformatSourcePython.bat)

Reply:
Formatted.

Reply:
Did you run the reformatpython.bat? It seems that line length limit is less than 120 in your format.

Reply:
Fixed format. The default values in my visual studio code was different.  

Also I found out that the https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/.style.yapf "base_on_style" has been changed from google to pep8. So I modified the instructions in the reformatpython.bat for consistency.